clients/src/main/java/org/apache/kafka/clients/admin/CreateTopicsResults.java

```diff
@@ -25,10 +25,10 @@
  * The result of newTopics.
  */
 @InterfaceStability.Unstable
-public class CreateTopicResults {
+public class CreateTopicsResults {
```

Comment:
I'm pretty neutral on `CreateTopicsResults` vs `CreateTopicsResult.`  Creating a `List<CreateTopicsResults>` is consistent with some of the other classes we have, though, right?